estate/models/estate_property.py

```diff
@@ -126,3 +126,16 @@ def _check_selling_price_and_expected_price(self):
                     "The selling price must be at leat 90% of the expected price! "
                     "You must reduce the expected price if you want to accept this offer."
                 )
+
+    @api.ondelete(at_uninstall=False)
+    def _unlink_if_property_status_not_new_canceled(self):
```

Comment:
This should be named `_unlink_if_<condition>` or `_unlink_except_<not_condition>` according to documentation.

```suggestion
    def _unlink_if_new_canceled(self):
```